tests/test_trackcompare.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import pandas as pd
+import numpy as np
+
+from fmskill.model import ModelResult
+from fmskill.observation import TrackObservation
+
+
+@pytest.fixture
+def observation():
+    fn = "tests/testdata/altimetry_NorthSea_20171027.csv"
+    df = pd.read_csv(fn, index_col=0, parse_dates=True)
+    return TrackObservation(df, item=2, name="alti")
+
+
+@pytest.fixture
+def modelresult():
+    fn = "tests/testdata/NorthSeaHD_extracted_track.dfs0"
+    return ModelResult(fn, name="HD")
+
+
+@pytest.fixture
+def comparer(observation, modelresult):
+    mr = modelresult
+    mr.add_observation(observation, item=2)
+    return mr.extract()
+
+
+def test_skill(comparer):
+    c = comparer
+    df = c.skill()
+
+    assert df.loc["alti"].n == 544
+
+
+def test_skill_vs_spatial_skill(comparer):
+    df = comparer.skill()  # to compare to result of .skill()
+    ds = comparer.spatial_skill()
+
+    assert df.loc["alti"].n == ds.n.values.sum()
+    assert ds.xBin.__len__() == 5
+    assert ds.yBin.__len__() == 5
+    # assert ds.coords._names == {"xBin","yBin"}  # TODO: Why return "observation" a by, when n_obs==1?
+
+
+def test_spatial_skill_bins(comparer):
```

Comment:
Where is the test that actually looks at the computed skill?

I am thinking something like `assert ds["bias"].sel(latitude=0.0, longitude=0.0, method='nearest') == pytest.approx(0.2)`

Reply:
I focused more on the binning here, as metrics is already tested. But yes, let's add this.

Reply:
https://github.com/DHI/fmskill/pull/38/commits/b811cea1205d3a52b025749146757fc6836fa1a1